proto/cosmos/distribution/v1beta1/tx.proto

```diff
@@ -48,7 +49,8 @@ service Msg {
 // MsgSetWithdrawAddress sets the withdraw address for
 // a delegator (or validator self-delegation).
 message MsgSetWithdrawAddress {
-  option (cosmos.msg.v1.signer) = "delegator_address";
+  option (cosmos.msg.v1.signer)            = "delegator_address";
+  option (cosmos.msg.legacy_amino.v1.name) = "cosmos-sdk/MsgSetWithdrawAddress";
```

Comment:
I think this has to be "MsgModifyWithdrawAddress" because it was always registered under a different (wrong?) name:

```
	cdc.RegisterConcrete(&MsgSetWithdrawAddress{}, "cosmos-sdk/MsgModifyWithdrawAddress", nil)
```